cmd/handler-api.go

```diff
@@ -272,6 +273,10 @@ func maxClients(f http.HandlerFunc) http.HandlerFunc {
 
 		globalHTTPStats.addRequestsInQueue(1)
 
+		if tc, ok := r.Context().Value(mcontext.ContextTraceKey).(*mcontext.TraceCtxt); ok {
+			tc.FuncName = "s3.MaxClients"
+		}
```

Comment:
Isn't this fixed via #16964

Reply:
> Isn't this fixed via https://github.com/minio/minio/pull/16964

This is for mc admin trace, not audit

Reply:
so this should be generalized? because it looks like other handlers also might not be visible.

Reply:
> so this should be generalized? because it looks like other handlers also might not be visible.

I don't think there is other handlers without a func name. We did not see this before because it is rare that a request is rejected by the server using the requests_max/requests_deadline feature.